service/src/main.rs

```diff
@@ -742,46 +748,64 @@ fn enclave_account<E: EnclaveBase>(enclave_api: &E) -> AccountId32 {
 	AccountId32::from(*tee_public.as_array_ref())
 }
 
-// Alice plays the faucet and sends some funds to the account if balance is low
-fn ensure_account_has_funds(api: &mut Api<sr25519::Pair, WsRpcClient>, accountid: &AccountId32) {
-	let alice = AccountKeyring::Alice.pair();
-	info!("encoding Alice's public 	= {:?}", alice.public().0.encode());
-	let alice_acc = AccountId32::from(*alice.public().as_array_ref());
-	info!("encoding Alice's AccountId = {:?}", alice_acc.encode());
-
-	let alice_free = api.get_free_balance(&alice_acc);
-	info!("    Alice's free balance = {:?}", alice_free);
-	let nonce = api.get_nonce_of(&alice_acc).unwrap();
-	info!("    Alice's Account Nonce is {}", nonce);
-
+// Alice plays the faucet and sends some funds to the account if balance is low and bootstrap_funds is true
```

Comment:
```suggestion
/// Alice transfers some balance to the parentchain account of the enclave if its balance is low and development mode is enabled.
```

Reply:
Actually, I think it would make sense to split the function `has_account_enough_funds` up into smaller functions. The current function name does not represent at all what the function is doing. In that case, a descriptive function comment might be rendered fully obsolete.

Reply:
For example, `has_account_enough_funds` should only check if the enclave has enough funds, nothing more.
After that check, you can then call a function that actually bootstraps the enclave, and name it accordingly.

Reply:
Actually I changed the min of the code and the comments as there is a 
`/// FIXME: needs some discussion (restructuring?)`
at the top of `start_worker.`
But yes, I will split the function `has_account_enough_funds`.